quarkus/server/distribution/LICENSE

```diff
@@ -2167,6 +2173,12 @@ License (from POM): Apache License 2.0 - https://www.apache.org/licenses/LICENSE
 
 --------------------------------------------------------------------------------
 
+Group: org.eclipse.persistence Name: eclipselink Version: 4.0.6
+Project URL: https://eclipse.dev/eclipselink/
+License: Eclipse Public License 2.0 - https://projects.eclipse.org/license/epl-2.0
+
+--------------------------------------------------------------------------------
+
 Group: org.glassfish.expressly Name: expressly Version: 5.0.0
```

Comment:
We don't. have a mention of Postgres here, should we include one?

Reply:
Actually, I added on admin distribution but not on server, fixing it now.